pytreex/core/node.py

```diff
@@ -273,13 +273,19 @@ def get_aligned_nodes(self):
         return ret
 
     def get_descendants(self, add_self=False, ordered=False,
-                        preceding_only=False, following_only=False):
+                        preceding_only=False, following_only=False, except_subtree=None):
```

Comment:
In Perl Treex it is called just "except", but this is not possible in Python. It is needed for reordering (and could be useful elsewhere).